mp4parse/src/lib.rs

```diff
@@ -2307,6 +2321,11 @@ pub fn read_mp4<T: Read>(f: &mut T) -> Result<MediaContext> {
             BoxType::MovieBox => {
                 context = Some(read_moov(&mut b, context)?);
             }
+            BoxType::MetadataBox => {
+                if let Some(ctx) = &mut context {
+                    ctx.metadata = Some(read_meta(&mut b));
+                }
+            }
```

Comment:
```suggestion
            #[cfg(feature = "meta-xml")]
                BoxType::MetadataBox => {
                if let Some(ctx) = &mut context {
                    ctx.metadata = Some(read_meta(&mut b));
                }
            }
```
Required based on putting `MediaContext.metadata` behind the feature flag